payload-files/remix/routes_test-local-variables-caught--event.json

```diff
@@ -166,17 +50,8 @@
         "version": "v20.11.1"
       },
       "trace": {
-        "data": {
-          "sentry.op": "function.remix.loader",
-          "sentry.origin": "auto.ui.remix"
-        },
-        "op": "function.remix.loader",
-        "origin": "auto.ui.remix",
         "parent_span_id": "[[ID3]]",
         "span_id": "[[ID4]]",
-        "tags": {
-          "name": "loader"
-        },
```

Comment:
The `loader` information is not included anymore. The error threw in a Remix loader